packages/core/elasticsearch/core-elasticsearch-client-server-internal/src/configure_client.test.ts

```diff
@@ -79,14 +80,14 @@ describe('configureClient', () => {
   });
 
   it('calls `parseClientOptions` with the correct parameters', () => {
-    configureClient(config, { logger, type: 'test', scoped: false, agentManager });
+    configureClient(config, { logger, type: 'test', scoped: false, agentManager, kibanaVersion });
 
     expect(parseClientOptionsMock).toHaveBeenCalledTimes(1);
     expect(parseClientOptionsMock).toHaveBeenCalledWith(config, false);
 
     parseClientOptionsMock.mockClear();
 
-    configureClient(config, { logger, type: 'test', scoped: true, agentManager });
+    configureClient(config, { logger, type: 'test', scoped: true, agentManager, kibanaVersion });
 
     expect(parseClientOptionsMock).toHaveBeenCalledTimes(1);
     expect(parseClientOptionsMock).toHaveBeenCalledWith(config, true);
```

Comment:
You need to adapt / fix the `expect(parseClientOptionsMock).toHaveBeenCalledWith` assertions